sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -85,6 +85,11 @@ export class ChatClient {
     const authPolicy = createCommunicationTokenCredentialPolicy(this.tokenCredential);
     this.client.pipeline.addPolicy(authPolicy);
 
+    (options as any).signalingClientOptions = {
+      ...(options as any).signalingClientOptions,
+      resourceEndpoint: this.endpoint,
```

Comment:
could this resource endpoint already exist in `signalingClientOptions` ?

Reply:
Checked, it is not existed in `signalingClientOptions`